conversion.py

```diff
@@ -59,11 +59,13 @@ def format_to_s2m2(env,env_folder,cfg_file):
         new_format["starts"].append(robots[i]["start"][:2]) # position just, no orientation
     for j in range(len(obstacles)):
         new_format["obstacles"].append(get_polytope(obstacles[j]["center"], np.array(obstacles[j]["size"]) / 2))
+   
     # add four obstacle to have the workspace limit
-    new_format["obstacles"].append(get_polytope([x_min-radius,(y_max-y_min)/2], [0.2,y_max-y_min]))
-    new_format["obstacles"].append(get_polytope([x_max+radius,(y_max-y_min)/2], [0.2,y_max-y_min]))
-    new_format["obstacles"].append(get_polytope([(x_max-x_min)/2,y_min-radius], [x_max-x_min,0.2]))
-    new_format["obstacles"].append(get_polytope([(x_max-x_min)/2,y_max+radius], [x_max-x_min,0.2]))
+    
+    new_format["obstacles"].append(get_polytope([x_min-radius-0.1,(y_max-y_min)/2], [0.2,(y_max-y_min)/2]))
+    new_format["obstacles"].append(get_polytope([x_max+radius+0.1,(y_max-y_min)/2], [0.2,(y_max-y_min)/2]))
+    new_format["obstacles"].append(get_polytope([(x_max-x_min)/2,y_min-radius-0.1], [(x_max-x_min)/2,0.2]))
+    new_format["obstacles"].append(get_polytope([(x_max-x_min)/2,y_max+radius+0.1], [(x_max-x_min)/2,0.2]))
```

Comment:
What are 0.2 and 0.1 here?

Reply:
I put 0.2 as the obstacle width, can be changed. And w/2 = 0.1 to specify the limits of Polytope.

Reply:
Thanks!